k8s/jobs/changeReplicationPasswordOnSecondary.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   generateName: change-replication-password-on-secondary-
+  namespace: adhoc-jobs
```

Comment:
Defining these here forces us to also create the namespace in production already, but I guess it won't hurt either.